library/systemd/src/lib/yast2/systemctl.rb

```diff
@@ -23,7 +23,9 @@ class << self
       def execute(command)
         command = SYSTEMCTL + command
         log.debug "Executing `systemctl` command: #{command}"
-        result = timeout(TIMEOUT) { SCR.Execute(Path.new(".target.bash_output"), command) }
+        result = ::Timeout.timeout(TIMEOUT) do
```

Comment:
Do we need full qualification here? `rescue Timeout::Error` below does not use it, it should be consistent.